client/my-sites/stats/hooks/use-location-views-query.ts

```diff
@@ -43,18 +42,6 @@ const useLocationViewsQuery = < T = StatsLocationViewsData >(
 		...options,
 		queryKey: [ 'stats', 'location-views', siteId, geoMode, JSON.stringify( finalQuery ) ],
 		queryFn: () => queryStatsLocationViews( siteId, geoMode, processQueryParams( finalQuery ) ),
-		select: ( data ) => {
```

Comment:
Don't normalize the data as part of the hook; instead, [normalize the data received from the hook](https://github.com/Automattic/wp-calypso/blob/324c673486f78c57a5b3ce344350927a13a144ca/client/my-sites/stats/features/modules/stats-locations/stats-locations.tsx#L173-L185). This is because the download CSV component also normalizes the data later down the line.

Reply:
Related to https://github.com/Automattic/wp-calypso/pull/103161/files#r2078147020.